src/engine/enginebuffer.cpp

```diff
@@ -185,6 +185,8 @@ EngineBuffer::EngineBuffer(QString group, UserSettingsPointer pConfig,
             this, SLOT(slotEjectTrack(double)),
             Qt::DirectConnection);
 
+    m_pTrackLoaded = new ControlPushButton(ConfigKey(m_group, "track_loaded"));
```

Comment:
Done and [documented on the wiki](http://mixxx.org/wiki/doku.php/developer_guide_control#read-only_controlobjects).